Source/carthage/Outdated.swift

```diff
@@ -7,6 +7,30 @@ import Curry
 
 /// Type that encapsulates the configuration and evaluation of the `outdated` subcommand.
 public struct OutdatedCommand: CommandProtocol {
+	enum UpdateAvailabilityAndApplicability: String {
+		case updatesAvailableAllApplicable =
+		"Will be updated to the newest version."
+		case updatesAvailableSomeApplicable =
+		"Will be updated, but not to the newest version because of the specified version in Cartfile."
+		case updatesAvailableNoneApplicable =
+		"Will not be updated because of the specified version in Cartfile."
+		case noUpdatesAvailable =
+		"Will not be updated: no updates available."
+
+		init(currentVersion current: PinnedVersion, applicableVersion applicable: PinnedVersion, latestVersion latest: PinnedVersion) {
+			switch (current, applicable, latest) {
+			case (current, applicable, latest) where current != latest && applicable == latest:
```

Comment:
This `switch`/`case` doesn't make sense: the `case`s are always true because they're matching what's in the `switch`. This should just be an `if`/`else`.